Dockerfile.rhel7

```diff
@@ -0,0 +1,11 @@
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
```

Comment:
BTW, the `Dockerfile` won't be used by anything, both CI and ART will use `Dockerfile.rhelX`. Would it make sense to have just Dockerfile and port ART to it?

Reply:
> Name this file .rhel8?

This was copied from https://github.com/openshift/aws-ebs-csi-driver-operator/blob/master/Dockerfile.rhel7#L1

> BTW, the Dockerfile won't be used by anything, both CI and ART will use Dockerfile.rhelX. Would it make sense to have just Dockerfile and port ART to it?

I guess it's OK, but it'll be different from the CSI driver which uses `Dockerfile.rhelX` because `Dockerfile` is taken.

Reply:
Renamed to `Dockerfile.rhel8`